### ISSUE TITLE
Add ability to hide tracks without removing them.

### ISSUE THREAD

**Issue by diekhans** [igvteam/igv#1283]
It would be useful to be able to hide a track to prevent it from taking up any screen real estate, without the need to reload it to display it again.  Especially great for creating sessions will all relevant data.

**Comment by jrobinso** [comment]
Noted.  How do you envision recovering the track if it's hidden?   Also, could you elaborate on "creating sessions with all relevant data"?

**Comment by diekhans** [comment]
Hi Jim

I am envisioning something similar to the Recorder Panels
windows to display tracks.  It would include all tracks, even
the visible ones.  Thus you could have one window to control the
visibility of both panels or tracks.

In fact, this could also include the Recorder Panels
functionality.  On one window, you could control visibility and
order of tracks and panels.

This is inspired by the UCSC Browser super tracks, where a panel
would be the super track with associated tracks (see attached
screen shot).

What I was hinting at with saving sessions with hidden tracks is being
able to save a set of tracks and share the with collaborators.

Right now, I am working on an analysis with a bigBed track,
three main BAM tracks and about 40 auxiliary BAM tracks.  I
share the saved XML session with a collaborator via github.

So instead of having a session with the four main tracks and
then sending them a list of URLs, I would send them a session
with the auxiliary tracks hidden.  This would make it super
easy to access the auxiliary, including turning them on an off.

If would be perfectly fine, actually desirable, to not load hidden
tracks until they are first open.
thanks!
Mark

![Screenshot 2023-02-12 at 9 49 59 PM](https://user-images.githubusercontent.com/110053/218382706-251dfc2a-33e0-4b7f-bab3-8bc22fb9d08c.png)

**Comment by jrobinso** [comment]
OK got it, thanks for the clarification.   Interesting idea.   We might think of this as a "view session" window.

**Comment by diekhans** [comment]
thank you!  Happy to help alpha test anything if you decide to pursue this.

**Comment by jrobinso** [comment]
This is quite an old issue, but FYI we are supporting UCSC track hubs in the next release, available now in the snapshot build.   With this solution you can create a hub with some or all tracks preselected.   Users load the "hub" by url which gives them a menu similar to the pic below.    This is emulating the UCSC menu style to the extent possible.   Creating a hub.txt file is quite easy, certainly easier than XML or even Json. 


<img width="1142" alt="Image" src="https://github.com/user-attachments/assets/b10ac55a-1b27-4cc0-ae38-97d162fe65f8" />

**Comment by maximilianh** [comment]
Yes, in UCSC hubs, track "sections" are called "composites" if the tracks
have the same type, or "superTracks", if they have different types and yes,
you can select/unselect the entire group with one click.
